Tangem/Modules/Onboarding/Wallet/WalletOnboardingViewModel.swift

```diff
@@ -381,15 +379,19 @@ class WalletOnboardingViewModel: OnboardingTopupViewModel<WalletOnboardingStep,
     @Published private var previewBackupState: BackupService.State = .finalizingPrimaryCard
     private var walletCreationType: WalletCreationType = .privateKey
 
-    private var tangemSdk: TangemSdk { tangemSdkProvider.sdk }
-    private var backupService: BackupService { backupServiceProvider.backupService }
+    private var tangemSdk: TangemSdk
+    private var backupService: BackupService
```

Comment:
вроде не меняется нигде
```suggestion
    private let tangemSdk: TangemSdk
    private let backupService: BackupService
```